web-api/src/app-public.ts

```diff
@@ -178,6 +181,25 @@ app.get('/public-api/judges', lambdaWrapper(getPublicJudgesLambda));
   );
 }
 
+/**
+ * Trial sessions
+ */
+{
+  app.get(
+    '/public-api/trial-sessions',
+    lambdaWrapper(getPublicTrialSessionsLambda),
+  );
+  app.get(
+    '/public-api/sections/:section/users',
+    lambdaWrapper(getUsersInSectionLambda),
+  );
+
```

Comment:
No reason for this space. I will remove, either in response to feedback on this PR or as part of a follow-up PR.